### DISCUSSION TITLE
Stocks Bundles

### DISCUSSION THREAD

**Discussion by sadortun** [GoPlan-Finance/goplan-app#45] [Ideas]
We all know Indexes, ETFs and Watchlists, the concept of a stock bundle goes a little bit further.

A stock bundle is a public/sharable watchlist that groups similar stocks toghether. For example:
- NoSQL Bundle ( contains MDB, ORCL ,.... ) 
- Big Tech Bundle (AMZN, GOOG, .... )
- US High dividends ( banks ... )
- ....

More ideas : 
- [ ] Bundles can include sub-bundles. 
- [ ] You can include bundles in your watchlist
- [ ] `My Bundles` list update automactly when you have all stocks from a bundle
- [ ] You can search for bundles coresponding to stocks you own ( ex: You own GOOG and AMZN, you should be able to find easely the  `Big Tech bundle`
- [ ] `My Watchlists` will group / highlight bundles

**Comment by mkitzmann** [comment]
How would you create these bundles? Would a user manually create a bundle and add stocks to it or would they be pre defined by certain parameters? So `Big Tech Bundle` would be defined by `sector = it, market-capitalisation > 1b`

**Reply by sadortun** [reply]
So, I see this in 3 types:

1) Market defined bundles (again, sorry for my terrible names ). Theses are your [S&P 500](https://www.slickcharts.com/sp500), ETFs composition Mutual founds, etc ...

This type can grow fast as we connect new data sources. / providers. 

---

2) Generated market bundles. Bundles that we create and maintain by some sort of criteria. "Oil compagnies" , Big Tech, high volatility stocks,  Today's 52wk lows, New Stocks & IPOs ... 

This will grow slowly and will require some sort of logic filter to be implemented by us.

---

3) User bundles / Watchlists*
Create your lists and share them.  "I WANT TO BUY", "New tech", "NoSQL databases" ...

I really think having a 3 type approach is extremely powerful and will allow users to share their insights, and get inspired by other.

---

 (*) For me the concept of bundle and watchlist is exactly the same. I like to use a different name since.i think the bundle is a bit more generic, as opposed to a watchlist which is a list a user made for his own private use.

**Reply by mkitzmann** [reply]
So, how about we have a public <> private toggle for watchlists. So you would have multiple watchlists, some for private use and some to share. It would work similiar to playlists in spotify.

So we could provide premade watchlists by differents metrics and show them to the users in a window similar to the `Browse` feature in spotify.

![DeepinScreenshot_select-area_20210311070637](https://user-images.githubusercontent.com/35574021/110743456-56bdf080-8238-11eb-8eef-a3dfde5e1709.png)

**Reply by mkitzmann** [reply]
Actually we could also have a `Home` section, where a user would see new watchlists by people he follows, popular watchlists, and his own watchlists next to performance information about his current portfolio. So this would be the start page for the application and the portfolio dashboard would be inside that portfolio.

![image](https://user-images.githubusercontent.com/35574021/110744024-4a866300-8239-11eb-8746-7431721d120e.png)
